controllers/v1/observations.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * name : observations.js
+ * author : Aman
+ * created-date : 22-Nov-2020
+ * Description : Observation related information.
+ */
+
+// Dependencies
+const observationsHelper = require(MODULES_BASE_PATH + "/observations/helper");
+
+module.exports = class Observations extends Abstract {
+    
+    constructor() {
+        super(schemas["observations"]);
+    }
+
+    /**
+  * @api {post} /kendra/api/v1/observations/update/:observationId Update Solutions
```

Comment:
@aman-tunerlabs - Do we really need these APIs in kendra ? Can it not be in samiksha?

Reply:
@aks30  Will make the changes in samiksha